packages/react/src/utils/constants.ts

```diff
@@ -3,6 +3,7 @@ export const PATIENT_CLICK_THRESHOLD = 500;
 export const DISABLED_TRANSITIONS_STYLE = { style: { transition: 'none' } };
 export const EMPTY_OBJECT = {};
 export const EMPTY_ARRAY = [];
+export const CLICK_TRIGGER_IDENTIFIER = 'data-click-trigger';
 
```

Comment:
Not sure if we should prefix here (`data-base-ui-*`) - `NavigationMenu.Trigger` also has a similar attribute

Reply:
OK, it makes sense to prefix it, then.